src/general/OptionsMenu.cs

```diff
@@ -600,6 +611,15 @@ private void SwitchMode(OptionsMode mode)
         }
     }
 
+    // Displays the current Resolution in options.
+    private void DisplayResolution()
+    {
+            var autoResolution = new LocalizedString("AUTO_RESOLUTION",
+                GetViewportRect().Size.x.ToString(CultureInfo.CurrentCulture),
+                GetViewportRect().Size.y.ToString(CultureInfo.CurrentCulture));
+            resolution.Text = autoResolution.ToString();
```

Comment:
```suggestion
            var resolution = GetViewportRect().Size;
            resolution.Text = string.Format(CultureInfo.CurrentCulture, TranslationServer.Translate("AUTO_RESOLUTION"), resolution.x, resolution.y);
```

I think something like that should work, might need slight tweaking for it to compile.